Client/mods/deathmatch/logic/lua/CLuaFunctionDefs.Util.cpp

```diff
@@ -12,27 +12,40 @@
 
 int CLuaFunctionDefs::GetValidPedModels(lua_State* luaVM)
 {
-    int iIndex = 0;
+    bool includeCustom = true;
+    CScriptArgReader argStream(luaVM);
+    argStream.ReadBool(includeCustom);
```

Comment:
is `includeCustom` enabled bu default? It looks like `argStream.ReadBool(includeCustom);` rewrites it with `false`.
i guess it should be `argStream.ReadBool(includeCustom, true);`

Reply:
ah yes it should be enabled by default, let me edit this